src/common/snippets/include/snippets/lowered/pass/init_live_ranges.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "pass.hpp"
+#include "snippets/generator.hpp"
+#include "snippets/lowered/reg_manager.hpp"
+
+namespace ov {
+namespace snippets {
+namespace lowered {
+namespace pass {
+
+/**
+ * @interface InitLiveRanges
+ * @brief Calculates live ranges of registers. This information will be used to assign registers and optimize ABI reg spills.
+ * @ingroup snippets
+ */
+class InitLiveRanges : public Pass {
+public:
+    OPENVINO_RTTI("InitLiveRanges", "Pass")
```

Comment:
```suggestion
    OPENVINO_RTTI("InitLiveRanges", "0", ov::snippets::lowered::pass::Pass)
```

Reply:
Updated here and in all other snippets::lowered passes.